sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -76,6 +76,13 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with SharedSQLContext
     sum
   }
 
+  private def getNumInMemoryTableScanExecs(plan: SparkPlan): Int = {
```

Comment:
we need a better name, this actually get in-memory table recursively, which is different from `getNumInMemoryRelations`